### PR TITLE
Add kv operations examples for dev-reorg

### PR DESCRIPTION
**Pull request by maria-robobug**
Supplements https://github.com/couchbase/docs-server/pull/2275 

Adding some basic examples that we can use in the `dev-reorg` server docs: https://issues.couchbase.com/browse/DOC-9205

### CODE REVIEW

File: modules/hello-world/examples/KvBulkHelloWorld.csx [hunk 1]
```diff
@@ -0,0 +1,130 @@
+// Run this using dotnet-script: https://github.com/filipw/dotnet-script
+//
+//      dotnet script KvBulkHelloWorld.csx
```
Comment (osfameron) [inline modules/hello-world/examples/KvBulkHelloWorld.csx:3]: woo dotnet script!

File: modules/hello-world/examples/KvBulkHelloWorld.csx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+//
+
+#r "nuget: CouchbaseNetClient, 3.2.5"
+
+using System;
+using System.Threading.Tasks;
+using Couchbase;
+using Couchbase.KeyValue;
+using Newtonsoft.Json.Linq;
```
Comment (osfameron) [inline modules/hello-world/examples/KvBulkHelloWorld.csx]: are you using Newtonsoft in this example?
(I see the next one has e.g. JArray and JProperty)

**Review by osfameron — APPROVED** [review]
👍

File: modules/hello-world/examples/KvBulkHelloWorld.csx (hunk 1 above)
Comment (maria-robobug) [reply modules/hello-world/examples/KvBulkHelloWorld.csx:3]: So much nicer running this way.

File: modules/hello-world/examples/KvBulkHelloWorld.csx (hunk 2 above)
Comment (maria-robobug) [reply modules/hello-world/examples/KvBulkHelloWorld.csx]: Only in the next example file, unnecessary import this, will remove 👍 .

**Pull request merged**